src/cryptonote_core/blockchain.cpp

```diff
@@ -132,6 +132,16 @@ static const struct {
 };
 static const uint64_t testnet_hard_fork_version_1_till = 624633;
 
+static const struct {
+  uint8_t version;
+  uint64_t height;
+  uint8_t threshold;
+  time_t time;
+} stagenet_hard_forks[] = {
+  // version 1 from the start of the blockchain
+  { 1, 1, 0, 1341378000 },
+};
+
```

Comment:
AFAIK, this is supposed to follow the monero forks at roughly the same time, otherwise it's just another testnet.

Reply:
I was wondering about this too. Should we add hard forks up to 6 in a rapid succession?

Reply:
Probably. I think having pre-rct txes present would be good too, since those may need to be tested too, so "rapid succession" might need to be not TOO rapid. I guess the people who have a use for this new network should mine on it, and make dummy txes to seed it with useful data.

Reply:
I added a comment on https://github.com/monero-project/monero/issues/3017 where at least one user of this network posted.